### DISCUSSION TITLE
US style event time format on button (am / pm)

### DISCUSSION THREAD

**Discussion by 35233larry** [add2cal/add-to-calendar-button#358] [Ideas / Requests]
Is it possible to change the style of the time on the button to US? Meaning am/pm instead of 09:00 / 18:00? A US user who looks at the button and sees 09:00 will not know if it's in the morning or afternoon.

**Comment by jekuer** [comment]
Fair point. Will look into this to see how easy it can be changed, bearing in mind that am/pm is not common outside most English speaking countries.

**Comment by jekuer** [comment]
Optimized with latest version 2.3
